anta/cli/__init__.py

```diff
@@ -55,11 +55,15 @@
     ),
     callback=setup_logging,
 )
+@click.option("--ignore-status", show_envvar=True, default=False, help="Always exit with success", is_eager=True)
+@click.option("--ignore-error", show_envvar=True, default=False, help="Only report failures and not errors", is_eager=True)
 def anta(ctx: click.Context, inventory: AntaInventory, **kwargs: Dict[str, Any]) -> None:
```

Comment:
Possible to do that:
```suggestion
def anta(ctx: click.Context, inventory: AntaInventory, ignore_status: bool, ignore_error: bool, **kwargs: Dict[str, Any]) -> None:
```